ticdc/integrate-confluent-using-ticdc.md

```diff
@@ -1,130 +1,264 @@
+    tiup ctl:v6.1.0 cdc changefeed create --pd="http://127.0.0.1:2379" --sink-uri="kafka://<broker_endpoint>/ticdc-meta?protocol=avro&replication-factor=3&enable-tls=true&auto-create-topic=true&sasl-mechanism=plain&sasl-user=<broker_api_key>&sasl-password=<broker_api_secret>" --schema-registry="https://<schema_registry_api_key>:<schema_registry_api_secret>@<schema_registry_endpoint>" --changefeed-id="confluent-changefeed" --config changefeed.conf
     ```
 
-## Integration procedures
-
-1. Save the following configuration into `jdbc-sink-connector.json`:
-
-    {{< copyable "" >}}
-
-    ```json
-    {
-      "name": "jdbc-sink-connector",
-      "config": {
-        "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector",
-        "tasks.max": "1",
-        "topics": "testdb_test",
-        "connection.url": "sqlite:test.db",
-        "connection.ds.pool.size": 5,
-        "table.name.format": "test",
-        "auto.create": true,
-        "auto.evolve": true
-      }
-    }
+    You need to replace the values of the following fields with those created or recorded in [Step 2. Create an access key pair](#step-2-create-an-access-key-pair):
+
+    - `<broker_endpoint>`
+    - `<broker_api_key>`
+    - `<broker_api_secret>`
+    - `<schema_registry_api_key>`
+    - `<schema_registry_api_secret>`
+    - `<schema_registry_endpoint>`
+
+    Note that you should encode `<schema_registry_api_secret>` based on [HTML URL Encoding Reference](https://www.w3schools.com/tags/ref_urlencode.asp) before replacing its value. After you replace all the preceding fields, the configuration file is as follows:
+
+    ```shell
+    tiup ctl:v6.1.0 cdc changefeed create --pd="http://127.0.0.1:2379" --sink-uri="kafka://xxx-xxxxx.ap-east-1.aws.confluent.cloud:9092/ticdc-meta?protocol=avro&replication-factor=3&enable-tls=true&auto-create-topic=true&sasl-mechanism=plain&sasl-user=L5WWA4GK4NAT2EQV&sasl-password=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" --schema-registry="https://7NBH2CAFM2LMGTH7:xxxxxxxxxxxxxxxxxx@yyy-yyyyy.us-east-2.aws.confluent.cloud" --changefeed-id="confluent-changefeed" --config changefeed.conf
     ```
 
-2. Create an instance of the JDBC sink connector by running the following command (assuming Kafka is listening on `127.0.0.1:8083`):
+    - Execute the configuration file.
 
-    {{< copyable "shell-regular" >}}
+        - If the file is successfully executed, information about the created changefeed, such as changefeed ID, is displayed, as shown below:
+
+            ```shell
+            Create changefeed successfully!
+            ID: confluent-changefeed
+            Info: {... changfeed info json struct ...}
+            ```
+
+        - If no result is returned after you execute the file, check the network connectivity between the server where you execute the file and Confluent Cloud. For details, see [Test connectivity to Confluent Cloud](https://docs.confluent.io/cloud/current/networking/testing.html).
```

Comment:
Execute the command, not configuration file.

Reply:
Fixed in [3fa1b2c](https://github.com/pingcap/docs/pull/9866/commits/3fa1b2c665c117963cd01ccdc9574a30475dada0)